packages/faceted-search/src/components/Badges/BadgeTags/BadgeTags.scss

```diff
@@ -0,0 +1,49 @@
+.fs-badge-tags-form {
+	:global(.tc-tooltip-body) {
+		margin: 1.2rem 0 0;
+		padding: 0.5rem 2rem 2rem;
+		overflow-y: auto;
+		background: #fcfcfc;
+		border-top: 0.1rem solid #ededed;
```

Comment:
Same here for the color
For the border width, do we want to keep 1px of to make it grow with the font ? I think it makes sense to keep it in pixel here